vms/vms/settings.py

```diff
@@ -129,10 +129,12 @@
 
 # Instead of sending out real email, during development the emails will be sent
 # to stdout, where from they can be inspected.
-if DEBUG:
-    EMAIL_HOST = config('EMAIL_HOST', default='localhost')
-    EMAIL_PORT = config('EMAIL_PORT', default=1025, cast=int)
-    EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
```

Comment:
keep this in comments for better understanding rather than just removing it completely